algorithms/CPlusPlus/Maths/prime-number.cpp

```diff
@@ -0,0 +1,23 @@
+#include<iostream>				//header
+using namespace std;
+int main() {					//main function
+	int num, i, a = 0;			//define variable
+	cin >> num;					//taking input from user
+	if(num == 2) {				//checking if condition for 2
+		cout << "Prime";
+	}
```

Comment:
No need to use this condition because when the for loop runs for **input:2** then It will not start because the condition to run the loop is num/2 (*where num is 2 and if 2/2 it gives 1*) and as 1 is less than *i* so loop stops

You should replace this loop with the condition
```C++
if(num<2)
{
a++;
}
```
> Just think that I enter 1 and this case run and increment i++ so the condition 
```C++
if(a == 0)
```
will not run and the else condition runs and print Non-Prime
> This case also handle non-negative number easily